deploy/charts/alluxio/templates/helpers/_alluxio-site-properties.tpl

```diff
@@ -23,6 +23,7 @@ alluxio.dora.client.ufs.root={{ .Values.dataset.path }}
 {{- range $key, $val := .Values.properties }}
 {{ printf "%v=%v" $key $val }}
 {{- end }}
+alluxio.cluster.name={{ .Release.Namespace }}-{{ .Release.Name }}
```

Comment:
can u give me an exampel of how this might look like if i do 
helm deploy "johndoe" ... ?

Reply:
`helm install alluxio -f <config-file> .` --> `alluxio.cluster.name=default-alluxio`
`helm install bob -f <config-file> -n namespace-bob` --> `alluxio.cluster.name=namespace-bob-bob`

Reply:
understood sounds good.